packages/account-v2/src/components/Dropzone/__tests__/Dropzone.spec.tsx

```diff
@@ -5,7 +5,7 @@ import Dropzone from '../Dropzone';
 
 describe('Dropzone', () => {
     const textPlainText = 'text/plain';
-    const dropZoneInputTestId = 'dt_dropzone-input';
```

Comment:
[Question]: What is the reason behind changing the naming convention of testids?

Reply:
@suisin-deriv You can check this doc https://wikijs.deriv.cloud/en/Frontend/development-guidelines/test_guide  for TestID naming convention

Reply:
Will follow the documentation provided and change it to `snake_case`